src/access/AccessControl.h

```diff
@@ -104,15 +105,15 @@ class AccessControl
 
         Entry() = default;
 
-        Entry(Entry && other) : mDelegate(other.mDelegate) { other.mDelegate = &mDefaultDelegate; }
+        Entry(Entry && other) : mDelegate(other.mDelegate) { other.mDelegate = &*mDefaultDelegate; }
```

Comment:
This is really not obvious syntax. Most average developers do not expect this.

Reply:
It's standard for things like smart pointers, std::optional etc...

I'm not too fussed either way... we can make `get()` public for use cases like this one, or we can even get rid of the operator overloads entirely and use `get()` everywhere to make it obvious at the call site that there's potentially more happening than just a plain variable access or pointer dereference.

Reply:
I have removed the `operator*` overload and made `get()` public, this should make call sites more obvious.